app/gui/src/model/module.rs

```diff
@@ -411,6 +411,7 @@ pub struct NodeMetadata {
     ///
     /// The methods may be defined for different types, so the name alone don't specify them.
     #[serde(default, deserialize_with = "enso_prelude::deserialize_or_default")]
+    // TODO: It is not used anywhere. Deprecate?
     pub intended_method: Option<MethodId>,
```

Comment:
Now all usages of the `intended_method` are removed. We are no longer using it to communicate with the engine, so this field should be probably removed or marked as deprecated.

Reply:
Marked it as deprecated and removed all usages from the code.